### DISCUSSION TITLE
parallels + wordpress + ray

### DISCUSSION THREAD

**Discussion by DimaMinka** [spatie/laravel-ray#93] [Q&A]
Hello
I try to run the ray on my MacBook with Big Sur. The bash version worked good. But from parallels (ubuntu 20.04), I can't see any output. 
Project stack:
Parallels + wordops
WordPress 5.5.3
WordPress plugin spatie-ray activated
PHP 7.4 + nginx + mysql

Checked with theme and plugin.

**Comment by freekmurze** [comment]
Ray runs locally on port 23517. I don't have experience with parallels, buy my guess is that you need to make sure that everything that gets sent to that part inside parallels gets forwarded to your local environment.
